com.unity.render-pipelines.high-definition/Documentation~/Compositor-User-Guide.md

```diff
@@ -1,15 +1,21 @@
 # Using the Graphics Compositor
-To open the Graphics Compositor window in the Unity Editor, select **Window > Render Pipeline > Graphics Compositor** from the menu. From this window, you can control the Graphics Compositor configuration. The first time you open the Graphics Compositor window, it automatically uses a default *"pass-through"* composition profile that forwards the output of the main [Camera](HDRP-Camera.md) to the final composed frame. You can edit this profile or you can load another one from disk. For information on the properties in the Compositor window, see [Compositor window](Compositor-User-Options.md).
+To open the Graphics Compositor window in the Unity Editor, select **Window > Render Pipeline > Graphics Compositor** from the menu. From this window, you can enable the Graphics Compositor and control its configuration. The first time you enable the Graphics Compositor, it automatically uses a default *"pass-through"* composition profile that forwards the output of the main [Camera](HDRP-Camera.md) to the final composed frame. You can add more [Layers](#layer-types) to this profile and edit their properties or you can load another profile from disk. For information on the properties in the Compositor window, see [Compositor window](Compositor-User-Options.md).
 
+## Alpha-based composition
+Many compositing operations require an alpha channel. To properly use all features of the graphics compositor, it is recomented (but not necessary) to:
+- Set the color buffer format for Rendering to 16-bit half float (R16G16B16A16)
+- Set the buffer format for post-processing to 16-bit half float (R16G16B16A16)
+
+Both options can be changed in the [HDRP Asset](HDRP-Asset). If the compositor detects a buffer format without an alpha channel it will display a warning. See also the documentation on [Alpha Output](Alpha-Output) for more details.
 
 ## Layer Types
 The Graphics Compositor tool typically handles two types of layers: 
-- **Composition Layers**: Which you define in the [Composition Graph](#composition-graph). The Composition Graph defines the number of layers and how to combine them but does not define each layer's content.
-- **Sub-layers**: Which you define in the Graphics Compositor window, in the [Render Schedule](#render-schedule) section. You use stack Sub-layers to define the content of a Composition Layer.
+- **Composition Layers**: The composition layers are the render targets (images) that are blended together to produce the final image. You can define them in the [Composition Graph](#composition-graph). The Composition Graph defines the number of layers and how to combine them but does not define each layer's content.
```

Comment:
The composition layers are the render targets (images) the compositor blends together to produce the final image. You can define them in the [Composition Graph](#composition-graph). The Composition Graph defines the number of layers and how to combine them but does not define each layer's content.